From 4aa5d70035caf62c3afe8ea977935c131ae71aa9 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Mon, 2 Feb 2015 14:21:31 +0800
Subject: [PATCH 1472/1543] MLK-10193: dts: Remove obsolete dts item
 csi1_v4l2_cap

commit a1d3b98ad9dbd53aea088855ae6da1251ce18980 from
git://git.freescale.com/imx/linux-2.6-imx.git

Remove csi1_v4l2_cap item from imx6sx-sdb-lcdif.dts file.
The item is not used in dts now.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
index 227f643..10da5dd 100644
--- a/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb-lcdif1.dts
@@ -9,10 +9,6 @@
 #include "imx6sx-sdb.dts"
 
 / {
-	csi1_v4l2_cap {
-		status = "disabled";
-	};
-
 	regulators {
 		reg_lcd_3v3: lcd-3v3 {
 			status = "okay";
-- 
1.7.5.4

